### PR TITLE
Create initial files and folders

### PR DESCRIPTION
**Pull request by rjharishabh**
## Initial files and folders created
* com_admin
    * sql
        * updates
            * mysql
                * **4.0.0-2021-06-08.sql**
* com_cookiemanager
    * **access.xml**
    * **config.xml**
    * **cookiemanager.xml**
    * forms
        * **filter_cookiemanager.xml**
    * services
        * **provider.php**
    * src
        * Controller
            * **DisplayController.php**
            * **CookiemanagerController.php**
        * Model
            * **CookiemanagerModel.php**
        * View
            * Cookiemanager
                * **HtmlView.php**
    * tmpl
        * cookiemanager
            * **default.php**
* language
    * en-GB
        * **com_cookiemanager.ini**
        * **com_cookiemanager.sys.ini**
     
I will update it

### Result after PR
cookiemanager

options

### CODE REVIEW

File: administrator/components/com_cookiemanager/cookiemanager.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension type="component" method="upgrade">
+	<name>com_cookiemanager</name>
+	<author>Joomla! Project</author>
+	<creationDate>June 2021</creationDate>
+	<copyright>(C) 2021 Open Source Matters, Inc.</copyright>
+	<license>GNU General Public License version 2 or later; see LICENSE.txt</license>
+	<authorEmail>admin@joomla.org</authorEmail>
+	<authorUrl>www.joomla.org</authorUrl>
+	<version>4.0.0</version>
```
Comment (chmst) [inline administrator/components/com_cookiemanager/cookiemanager.xml:10]: Version will be adapted

Comment (rjharishabh) [reply administrator/components/com_cookiemanager/cookiemanager.xml:10]: version will be 4.1.0 or anything else

Comment (rjharishabh) [reply administrator/components/com_cookiemanager/cookiemanager.xml:10]: Sorry, I didn't understand @chmst what you want to say.

Comment (rjharishabh) [reply administrator/components/com_cookiemanager/cookiemanager.xml:10]: IIRC in one PR, Brian talks about this version adaption, which means the version will be automatically adapted according to the version used by the admin, I don't know how to do it, and I unable to find that PR.
@chmst I think you are talking about that.

File: administrator/components/com_cookiemanager/cookiemanager.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension type="component" method="upgrade">
+	<name>com_cookiemanager</name>
+	<author>Joomla! Project</author>
+	<creationDate>June 2021</creationDate>
+	<copyright>(C) 2021 Open Source Matters, Inc.</copyright>
```
Comment (jeckodevelopment) [inline administrator/components/com_cookiemanager/cookiemanager.xml:6]: The Copyright notice should be:
`<copyright>(C) 2005 - 2021 Open Source Matters. All rights reserved.</copyright>`

Comment (nibra) [reply administrator/components/com_cookiemanager/cookiemanager.xml:6]: Nope, that's not right The year is the year of first appearance in the public. We made a great effort to change that format to match the standard. So no need to change here.

Comment (jeckodevelopment) [reply administrator/components/com_cookiemanager/cookiemanager.xml:6]: Thanks @nibra

File: administrator/language/en-GB/com_cookiemanager.ini [hunk 3]
```diff
@@ -0,0 +1,11 @@
+; Joomla! Project
+; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+COM_COOKIEMANAGER="Cookie Manager"
+COM_COOKIEMANAGER_XML_DESCRIPTION="Cookie manager to manage cookies."
```
Comment (chmst) [inline administrator/language/en-GB/com_cookiemanager.ini:7]: At the end, the language keys mus be in alphabetical order.

Comment (rjharishabh) [reply administrator/language/en-GB/com_cookiemanager.ini:7]: Ok, at the end, I will sort it in the alphabetical order.

**Pull request closed without merging**